e2e/autoscript-apps/project.json

```diff
@@ -9,6 +9,7 @@
       "outputs": ["{options.outputPath}"],
       "options": {
         "outputPath": "dist/e2e/autoscript-apps",
+        "compiler": "swc",
```

Comment:
swc is a faster compiler but does not typecheck. This will compile with swc, typechecking is still tsc.